fsserver/fsclient.c

```diff
@@ -17,12 +17,12 @@
  * You should have received a copy of the GNU General Public License
  * along with this program. If not, see <http://www.gnu.org/licenses/>.
  */
-#define _GNU_SOURCE
```

Comment:
This deleted line seems to be needed on FSL10 amd64.

Reply:
Ah that landed in there too early, I have moved that to the fsserver Makefile on another branch. 

Reverted for now.